multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -34,7 +34,7 @@ const accounts = [...keys(oracleMnemonics), 'lp'];
 const contractName = 'fastUsdc';
 const contractBuilder =
   '../packages/builders/scripts/fast-usdc/init-fast-usdc.js';
-const LP_DEPOSIT_AMOUNT = 10_000_000n;
+const LP_DEPOSIT_AMOUNT = 8_000n * 10n ** 6n;
```

Comment:
we have the 10^6 as a constant somewhere but this is clear enough. better than the literal